include/client/templates/sidebar.tmpl.php

```diff
@@ -20,7 +20,7 @@
 <?php } ?>
         <div class="content"><?php
     $faqs = FAQ::getFeatured()->select_related('category')->limit(5);
-    if ($faqs->all()) { ?>
+    if ($cfg->isKnowledgebaseEnabled() && $faqs->all()) { ?>
```

Comment:
I think faqs query should be within if statement so we don't want a trip to db on each page when when knowledge base is disabled. 

```php

if ($cfg->isKnowledgebaseEnabled() 
  && ($faqs = FAQ::getFeatured()->select_related('category')->limit(5))
  && $faqs->all()) { ?>

```